querydsl-apt/src/main/java/com/querydsl/apt/TypeElementHandler.java

```diff
@@ -43,7 +43,7 @@
  * @author tiwe
  *
  */
-final class TypeElementHandler {
+public class TypeElementHandler {
```

Comment:
Why the change?

Reply:
In order to customize it - https://github.com/infobip/infobip-spring-data-querydsl/pull/9/files#diff-3ecd73bd5e0c2a2d7e3360d04428617f
I need a hybrid approach between what sql maven plugin and jpa annotation processor do.

Reply:
The only alternative would be to duplicate 80% of the code.